bokehjs/gulp/tasks/generate_defaults.py

```diff
@@ -104,8 +104,5 @@ def output_defaults_module(filename, defaults):
 
 output_defaults_module(filename = os.path.join(dest_dir, 'common/defaults.coffee'),
                        defaults = all_json)
-output_defaults_module(filename = os.path.join(dest_dir, 'widget/defaults.coffee'),
+output_defaults_module(filename = os.path.join(dest_dir, 'models/widgets/defaults.coffee'),
```

Comment:
Sure, another PR is fine. I'm not sure about those locations, maybe just all of them at the top level is preferable, named for the modules they govern:
```
bokeh.coffee
bokeh-widgets.coffee
bokeh-compiler.coffee
```
etc.